packages/components/inputs/select-input/src/select-input.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import SelectInput from './select-input';
+import { iconArgType } from '@/storybook-helpers';
+
+const meta: Meta<typeof SelectInput> = {
+  title: 'Form/Inputs/SelectInput',
+  component: SelectInput,
+  argTypes: {
+    iconLeft: iconArgType,
+    'aria-label': { control: { type: 'text' } },
+    'aria-labelledby': { control: { type: 'text' } },
+    'aria-invalid': { control: { type: 'boolean' } },
+    'aria-errormessage': { control: { type: 'text' } },
+    backspaceRemovesValue: { control: { type: 'boolean' } },
+    controlShouldRenderValue: { control: { type: 'boolean' } },
+    filterOption: { type: 'function' },
+    id: { control: { type: 'text' } },
+    inputValue: { control: { type: 'text' } },
+    containerId: { control: { type: 'text' } },
+    isClearable: { control: { type: 'boolean' } },
+    isDisabled: { control: { type: 'boolean' } },
+    isOptionDisabled: { type: 'function' },
+    isMulti: { control: { type: 'boolean' } },
+    isSearchable: { control: { type: 'boolean' } },
+    menuIsOpen: { control: { type: 'boolean' } },
+    maxMenuHeight: { control: { type: 'number' } },
+    menuPortalTarget: { control: false },
+    menuShouldBlockScroll: { control: { type: 'boolean' } },
+    closeMenuOnSelect: { control: { type: 'boolean' } },
+    name: { control: { type: 'text' } },
+    noOptionsMessage: { type: 'function' },
+    placeholder: { control: { type: 'text' } },
+    tabIndex: { control: { type: 'number' } },
+    tabSelectsValue: { control: { type: 'boolean' } },
+    value: { control: false },
```

Comment:
It should have been a `StoryFn` due to the state, not a `StoryObj`. Must have slipped through. Thanks for catching. I [added state](https://github.com/commercetools/ui-kit/pull/2882/commits/3641ca90fcbbbb8f3682448c4dec76beb4582d29) and also made sure it can handle a switch back and forth to/from `isMulti`